Dockerfile

```diff
@@ -0,0 +1,75 @@
+# Build with: docker build --ulimit nofile=65535:65535 . -t custom-renovate
+# Run with: docker run --rm <additional args> custom-renovate renovate
+
+FROM quay.io/fedora/fedora:40-x86_64
+LABEL description="Mintmaker - Renovate custom image" \
+      summary="Mintmaker basic container image - a Renovate custom image" \
+      maintainer="EXD Rebuilds Guild <exd-guild-rebuilds@redhat.com >"
+
+ARG RENOVATE_VERSION=37.407.1-custom
+
+# Using OpenSSL store allows for external modifications of the store. It is needed for the internal Red Hat cert.
+ENV NODE_OPTIONS=--use-openssl-ca
+
+RUN dnf update -y && \
+    dnf install -y \
+        git \
+        python3-dnf \
+        python3-pip \
+        python3.11 \
+        python3.10 \
+        python3.9 \
+        python3.8 \
+        nodejs \
+        npm \
+        skopeo \
+        podman && \
+    dnf clean all && \
+    rpm --install --verbose \
+        https://github.com/tektoncd/cli/releases/download/v0.35.1/tektoncd-cli-0.35.1_Linux-64bit.rpm
+
+# Add renovate user and switch to it
+RUN useradd -lms /bin/bash -u 1001 renovate
+
+WORKDIR /home/renovate
+USER 1001
+
+# Enable renovate user's bin dirs,
+#   ~/.local/bin for Python executables
+#   ~/node_modules/.bin for renovate
+ENV PATH="/home/renovate/.local/bin:/home/renovate/node_modules/.bin"
```

Comment:
```suggestion
ENV PATH="/home/renovate/.local/bin:/home/renovate/node_modules/.bin:$PATH"
```

Reply:
This causes the npm not found issue

Reply:
or `ENV PATH="/home/renovate/.local/bin:/home/renovate/node_modules/.bin:${PATH}"` like in the other Dockerfile, don't know if there's a syntax difference, but just $PATH seems to work for me